python/pyspark/ml/clustering.py

```diff
@@ -323,7 +323,7 @@ def trainingCost(self):
         return self._call_java("trainingCost")
 
 
-class KMeansModel(JavaModel, JavaMLWritable, JavaMLReadable):
+class KMeansModel(JavaModel, GeneralJavaMLWritable, JavaMLReadable):
```

Comment:
I see, this is all we need because the Scala implementation already implements `GeneralMLWritable`? sounds good as this is what LinearRegressionModel does too.